app/Helpers/UploadFile.php

```diff
@@ -1,23 +1,22 @@
 <?php
+
 namespace ChopBox\helpers;
+
 use Cloudder;
 
 class UploadFile {
 
-
-/*
-  uploads a file to cloudinary using the clouder helper facade and returns
-  the result of the upload to the user or false if it fails;
-*/
-
+  /*
+   * uploads a file to cloudinary using the clouder helper facade and returns
+   * the result of the upload to the user or false if it fails;
+   */
   public function uploadFile($file) {
-
-    if($file != NULL) {
+    if ($file != NULL) {
```

Comment:
I think you should do a better check here . Use `isset()` and `is_null` to do the check.